Sources/KeyboardShortcuts/KeyboardShortcuts.swift

```diff
@@ -193,6 +193,8 @@ public enum KeyboardShortcuts {
 	}
 
 	static func userDefaultsSet(name: Name, shortcut: Shortcut) {
+		userDefaultsRemove(name: name)
```

Comment:
It should be placed after line 200.

And thinking more about this, I think it's enough to just call `unregister(shortcut)`, since `UserDefaults.standard.removeObject` (in the `userDefaultsRemove` method) is moot since we'll set the key right away afterwards.

Reply:
So I would make it:

```swift
if let oldShortcut = userDefaultsGet(name: name) {
	unregister(oldShortcut)
}

register(shortcut)
UserDefaults.standard.set(encoded, forKey: userDefaultsKey(for: name))
userDefaultsDidChange(name: name)
```

Can you do that and test that it still works fine in your app?

Reply:
I've just updated, and that works in my app, thanks.